manifests/s/SomePythonThings/WingetUIStore/1.6.2/SomePythonThings.WingetUIStore.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created with WinGet Releaser v2 using Komac v1.0.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: SomePythonThings.WingetUIStore
+PackageVersion: 1.6.2
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: user
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2023-02-27
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/marticliment/WingetUI/releases/download/1.6.2/WingetUI.Installer.exe

```

Comment:
@marticliment Can you update this PR and reopen it? URL should be `https://github.com/marticliment/WingetUI/releases/download/1.6.2-r2/WingetUI.Installer.exe`

Reply:
Done!